composer.json

```diff
@@ -13,6 +13,9 @@
     }
   ],
   "require": {
+    "php": "~5.6 || >=7.0",
```

Comment:
Dit mag eigenlijk >=7.1 worden, 5.6 ondersteunen we niet langer (unit tests gebeuren ook niet meer tegen die versie).